src/app/productBrowse/tests/productBrowse.spec.js

```diff
@@ -41,9 +39,9 @@ describe('Component: ProductBrowse', function () {
             injector.invoke(productsState.resolve.Parameters);
             expect(ocParametersService.Get).toHaveBeenCalled();
         });
-        it('should resolve ProductList', function () {
+        xit('should resolve ProductList', function () {
             injector.invoke(productsState.resolve.ProductList);
```

Comment:
yeah, i noticed the same. Lets just add a function (can store it in global.spec.js) that does something similar to angular.copy so that global constants aren't mutated.